src/roman-numerals/index.js

```diff
@@ -4,6 +4,23 @@
  * @param {string} roman The all-caps Roman numeral between 1 and 3999 (inclusive).
  * @returns {number} The decimal equivalent.
  */
-function romanToDecimal(roman) {}
+function romanToDecimal(roman) {
+  
+  roman = roman.toUpperCase()
+    var rom = {"I":1, "V":5, "X":10, "L":50, "C":100, "D":500, "M":1000, "IV":4, "IX":9, "XL":40, "XC":90, "CD":400, "CM":900};
```

Comment:
Prefer `let` or `const` for variable declaration.

Reply:
Noted.